domain/model/mesh_results.py

```diff
@@ -49,11 +49,30 @@ class MeshRow:
     columns: List[MeshColumn]
 
 
+class Agents:
+    def __init__(self, rows: List[MeshRow]) -> None:
+        self._rows = rows
+
+    def get_id(self, alias: str) -> AgentID:
+        for row in self._rows:
+            if row.agent_alias == alias:
+                return row.agent_id
+        return AgentID()
+
+    def get_alias(self, id: AgentID) -> str:
+        for row in self._rows:
+            if row.agent_id == id:
+                return row.agent_alias
+        return f"[agent_id={id} not found]"
+
+
 class MeshResults:
     """Internal representation of Mesh Test results; independent of source data structure like http or grpc synthetics client"""
 
-    def __init__(self) -> None:
+    def __init__(self, timestamp: datetime = datetime.utcnow()) -> None:
```

Comment:
```suggestion
    def __init__(self, timestamp: datetime = datetime.now(timezone.utc)) -> None:
```

It seems to be safer to have timezone info in the timestamp. Even if it works now, timestamp might have other usages in the future.

See: https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow

Reply:
Done